### DISCUSSION TITLE
Rationale for esoteric trigonometry functions

### DISCUSSION THREAD

**Discussion by ghalimi** [stdlib-js/stdlib#376] [Q&A] [answered]
What is the rationale for including esoteric yet rather basic trigonometry functions like [Arcversine](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/aversin)? These functions have the exact same [implementation](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/aversin/lib/aversin.js#L51) as one would write directly, therefore performance cannot be the motivation. Is it just a pure convenience?

**Comment by kgryte** [comment, marked as answer]
The motivation is straightforward: to name common mathematical expressions. That they are currently implemented according to textbook definitions (in most cases) is an implementation detail subject to change. For example, the error functions are an example where implementations do not follow textbook formulas. In the function, we may update the implementations for better accuracy, perf, or a combination of both, depending on time and funding.

**Reply by ghalimi** [reply]
That makes sense. Thank you.
